cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -1,12 +1,16 @@
 {% set base_metrics_memory = "140Mi" -%}
 {% set metrics_memory = base_metrics_memory -%}
+{% set base_metrics_cpu = "80m" -%}
+{% set metrics_cpu = base_metrics_cpu -%}
 {% set base_eventer_memory = "190Mi" -%}
 {% set eventer_memory = base_eventer_memory -%}
 {% set metrics_memory_per_node = 4 -%}
+{% set metrics_cpu_per_node = 0.5 -%}
 {% set eventer_memory_per_node = 500 -%}
 {% set num_nodes = pillar.get('num_nodes', -1) -%}
 {% if num_nodes >= 0 -%}
   {% set metrics_memory = (200 + num_nodes * metrics_memory_per_node)|string + "Mi" -%}
+  {% set metrics_cpu = (80 + num_nodes * metrics_cpu_per_node)|string + "m" -%}
```

Comment:
Note that the limits were used for scheduling, but not strictly enforced (ContainerVM didn't support it). The new GCI nodes should be enforcing it, so this may need to be checked and perhaps revised.